### PR TITLE
Scroll progress bar

### PR DESCRIPTION
**Pull request by emilankersen**

### CODE REVIEW

File: patterns/01-atoms/scroll-progress-bar/novicell.scroll-progress-bar.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * @author Emil Ankersen (EAN)
+ * @requires https://github.com/jeremenichelli/scrollProgress
+ */
+
+var novicell = novicell || {};
+
+novicell.scrollProgressBar = novicell.scrollProgressBar || new function () {
+
+    this.init = function () {
+        console.log('scrollProgressBar');
```
Comment (Dan9boi) [inline patterns/01-atoms/scroll-progress-bar/novicell.scroll-progress-bar.js]: should we remove this `console.log()`?

File: patterns/01-atoms/scroll-progress-bar/scroll-progress-bar.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.progress-bar {
```
Comment (Dan9boi) [inline patterns/01-atoms/scroll-progress-bar/scroll-progress-bar.css]: I think we should have variables for the color, and height, in the top, declared in `:root`, with fallbacks.
Can you add that?

File: patterns/04-pages/article-with-progress-bar/article-with-progress-bar.config.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "title": "Article with progress bar",
```
Comment (Dan9boi) [inline patterns/04-pages/article-with-progress-bar/article-with-progress-bar.config.json]: Change the name to just "Article"

File: patterns/01-atoms/scroll-progress-bar/novicell.scroll-progress-bar.js (hunk 1 above)
Comment (Dan9boi) [reply patterns/01-atoms/scroll-progress-bar/novicell.scroll-progress-bar.js]: Also remove the console.log's on the slider pattern!

**Comment by ottzen** [comment]
I fixed console logs and root declarations.

**Pull request merged**